app-misc/pwsafe/pwsafe-0.2.0-r5.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="A Password Safe compatible command-line password manager"
+HOMEPAGE="http://nsd.dyndns.org/pwsafe/"
```

Comment:
Have you checked that url, both for homepage and src_uri?
curl --head https://nsd.dyndns.org/pwsafe/
curl: (28) Failed to connect to nsd.dyndns.org port 443: Connection timed out

curl --head dyndns.org/
HTTP/1.1 301 Moved Permanently
Date: Tue, 01 Sep 2020 07:19:14 GMT
Server: Apache
Location: http://www.dyndns.com/
Connection: close
Content-Type: text/html; charset=iso-8859-1

Reply:
I know that site is dead. I don’t have any other homepage though. The author has a Github account, and pwsafe is there, but the Github project doesn’t have the 0.2.0 tarball or anything that resembles it by name. It’s a bit of a mess, but as long as the Gentoo mirror network still has the 0.2.0 tarball, it can still be installed, and I don’t really see a better option. Unless you suggest something else?

Reply:
I think _something_ is good because otherwise folks will end up filing bugs about it. Maybe just add the Github project? Or you could archive whatever you can find yourself on Github?

Reply:
The author has finally replied and said he does not intend to bring the site back, so I have changed `HOMEPAGE` to his Github site. He intends to make a release at some point but has not done so yet, so for now I have pointed `SRC_URI` at `web.archive.org`, which snapshotted the original tarball, so at least `SRC_URI` is now fetchable.